cosmos/migrations/0001_initial.py

```diff
@@ -0,0 +1,73 @@
+
+from cosmos.models.user.constants import DEPARTMENTS, NATIONALITIES, PROGRAMS
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("auth", "0011_update_proxy_permissions"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Board",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("description", models.TextField(blank=True)),
+                ("period_from", models.DateField()),
+                ("period_to", models.DateField()),
+                ("pretix_organizer", models.CharField(max_length=20)),
+                ("group", models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to="auth.Group")),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Profile",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("department", models.CharField(choices=[(name, name) for name in DEPARTMENTS], max_length=100,),),
+                ("program", models.CharField(choices=[(name, name) for name in PROGRAMS], max_length=100,),),
+                (
+                    "nationality",
+                    models.CharField(choices=[(country, country) for country in NATIONALITIES], max_length=100,),
+                ),
+                (
+                    "gender",
+                    models.CharField(
+                        choices=[("Male", "Male"), ("Female", "Female"), ("Other", "I'd rather not say")], max_length=10
+                    ),
+                ),
+                ("tue_id", models.CharField(max_length=25, verbose_name="TU/e Number")),
+                ("card_number", models.CharField(max_length=25)),
+                ("key_access", models.CharField(default="No", max_length=3)),
+                (
+                    "member_type",
+                    models.CharField(
+                        choices=[(status, status) for status in ["Member", "Pending", "Rejected"]],
+                        default="Pending",
+                        max_length=50,
+                    ),
+                ),
+                (
+                    "user",
+                    models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Committee",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("description", models.TextField(blank=True)),
+                ("pretix_token", models.CharField(max_length=64)),
+                ("board", models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to="cosmos.Board")),
```

Comment:
What do you want to do with this link?

Reply:
I was thinking one Committee was connected to one board, that way it'll be easier to group by year. If you had another thing in mind, feel free to change

Reply:
thing is afaik, committees are not really connected to a board specifically, or change every year like board, so i feel it would only create more difficulty in managing everything

Reply:
so, should I be removing it entirely? or rework the models somehow

Reply:
@bcvandendool Please let me know if there should be any more changes to the model before I makemigrations and squash

![image](https://user-images.githubusercontent.com/19555626/88669071-b4eff600-d0e3-11ea-8e3b-a4188b11b362.png)